README.md

```diff
@@ -140,6 +140,21 @@ If you come across any problems or would like to get in touch:
 * For bug reports or feature requests,
   [have a look at the GitHub issues or create a new one](https://github.com/balena-io/balena-cli/issues/).
 
+## Deprecation policy
```

Comment:
In addition to appearing in the README file on GitHub and the npm registry, I think we would like this deprecation policy to appear on the docs site too: https://www.balena.io/docs/reference/balena-cli/
This is achieved by automatically adding this README section to the `doc/cli.markdown` file (in this repo), which in turn is achieved by adding the following line of code to the following file:
```
mdParser.getSectionOfTitle('Deprecation policy'),
```
https://github.com/balena-io/balena-cli/blob/v11.32.15/automation/capitanodoc/capitanodoc.ts#L162